project_app/components/Header.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import Button from "@mui/material/Button";
+import IconButton from "@mui/material/IconButton";
+import ButtonGroup from "@mui/material/ButtonGroup";
+import { Container, AppBar, Typography } from "./styled";
+import Image from "next/image";
+
+function Header() {
+  return (
+    <>
+      <AppBar>
+        <Container maxWidth="xl">
+          <div style={{ position: "absolute", marginLeft: "700px" }}>
+            <Button>Login</Button>
+            <Button>Join Us!</Button>
+          </div>
+          <Image
+            src="/logo.svg"
+            width={227.77}
+            height={150}
+            left={50}
+            right={50}
+            alt="Ain't Board Logo"
+          />
+
+          <input
+            placeholder="Search Board Game"
+            InputProps={{
+              endAdornment: (
+                <Image
+                  src="/icons8.svg"
+                  width={227.77}
+                  height={150}
+                  alt="image"
+                />
+              ),
+            }}
+          />
+          <div>
+            <IconButton label="Events">
+              <p>Events</p>
+            </IconButton>
+            <IconButton label="FAQ">
+              <p>FAQ</p>
+            </IconButton>
+            <IconButton label="About Us">
+              <p>About Us</p>
+            </IconButton>
+            <IconButton label="Game night">
+              <p>Game Night</p>
+            </IconButton>
+          </div>
+        </Container>
+        <>
+          <Typography>Interactive BoardGame Community</Typography>
+
+          <div
+            style={{
+              position: "absolute",
+              width: "629px",
+              height: "471px",
+              top: "215px",
+              left: "0",
+            }}
+          >
+            <Image src="/bella.png" fill="absolute" alt="image" />
```

Comment:
lets have appropriate naming for the filenames

Reply:
updated filenames